### DISCUSSION TITLE
Refresh Interval?

### DISCUSSION THREAD

**Discussion by pwelker** [mampfes/ha_freeair_connect#9] [Q&A]
Hi, is there a default refresh interval? Or do I always need to refresh via calling the service freeair_connect.fetch_data via automation?

**Comment by pwelker** [comment]
Seems it's 1 hour. Is there an option to change it?

**Comment by mampfes** [comment]
The integration polls data every 10 minutes from the webpage (this is what we could change). But the webpage is updated less frequently (here we can't do anything).

So, there is nothing we could do ....

**Reply by pwelker** [reply]
OK, thanks. I am just asking because the LAST_TIMESTAMP value only changes once in an hour. But that helps.
